conversiontracerbullet/src/test/java/org/stanford/AuthDBPropertiesTest.java

```diff
@@ -28,23 +29,24 @@
 
     private AuthDBProperties authProps;
     private Properties serverConf;
-    private String propertyFile;
+    private String serverConfResourceName = "/server.conf";
     private File serverConfFile;
     private File tmpDir;
 
     @Before
     public void setUp() throws IOException, InvocationTargetException, IllegalAccessException, NoSuchMethodException, NoSuchFieldException {
+        tmpDir = createTempDirectory("AuthDBPropertiesTest_").toFile();
         authProps = new AuthDBProperties();
-        // Use private method to get the server.conf file path
-        Field f = AuthDBProperties.class.getDeclaredField("propertyFile");
-        f.setAccessible(true);
-        propertyFile = (String) f.get(authProps);
-        serverConfFile = new File(propertyFile);
-        // Use private method to get the server.conf properties
-        Method m = AuthDBProperties.class.getDeclaredMethod("loadDataSourceProperties", String.class);
+
+        Method m = AuthDBProperties.class.getDeclaredMethod("loadPropertyResource");
```

Comment:
I'm ok leaving this as is;  

It's true there is a  maintenance cost incurred that if you change the data being used for tests, you have to change the hardcoded data in the tests ... but in my experience it is still better to test for actual values than test a method's results against ... the same method's results.